data-serving/data-service/test/model/case.test.ts

```diff
@@ -38,3 +38,15 @@ describe('validate', () => {
         return new Case(fullModel).validate();
     });
 });
+
+describe('custom instance methods', () => {
+    it('equalsJSON returns true for identical case', () => {
+        const c = new Case(fullModel);
+        expect(c.equalsJSON(fullModel)).toBe(true);
+    });
+    it('equalsJSON returns false for differing case', () => {
```

Comment:
please add a test where revision metadata or another irrelevant field is changed and expect equalsJSON to be true

Reply:
SG, done.